Gemfile

```diff
@@ -3,7 +3,7 @@ gemspec
 
 gem 'rest-client', :github => 'opscode/rest-client'
 
-gem 'chef-pedant', :github => 'opscode/chef-pedant', :tag => '1.0.46'
+gem 'oc-chef-pedant', :github => 'opscode/oc-chef-pedant', :tag => '2.0.0'
```

Comment:
Any higher version of `chef-pedant` or `oc-chef-pedant` fail on the rake task.
It looks like newer versions of pedant would require `lib/chef_zero/endpoints/actors_endpoint.rb` to change the JSON to `{ "chef_key": { "private_key": "", "public_key": "" } }`.

Unfortunately I'm not 100% sure of the semantics of the real Chef 12 server with respect to this API.